ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -395,21 +395,49 @@ router bgp {{ router_bgp.as }}
 {%                      elif router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is defined and router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is not none %}
       neighbor {{ neighbor }} peer group {{ router_bgp.vrfs[vrf].neighbors[neighbor].peer_group }}
 {%                      endif %}
+{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].local_as is arista.avd.defined and router_bgp.vrfs[vrf].neighbors[neighbor].local_as is number %}
+      neighbor {{ neighbor }} local-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].local_as }} no-prepend replace-as
```

Comment:
Should we not have options for `no-prepend` and `replace-as` since `eos_cli_config_gen` does not provide any abstraction: EOS CLI should reflect what YAML file describe?

Reply:
AFAIK no-prepend and replace-as are mandatory parameters. Can't use local-as without them. 
![image](https://user-images.githubusercontent.com/50451665/105395630-3d78ca80-5c1f-11eb-9a0f-4161fa0b7cee.png)

Reply:
Thanks. Missed that one